src/bootstrap/src/core/config/tests.rs

```diff
@@ -352,3 +355,8 @@ fn parse_rust_std_features_empty() {
 fn parse_rust_std_features_invalid() {
     parse("rust.std-features = \"backtrace\"");
 }
+
+#[test]
+fn parse_jobs() {
+    assert_eq!(parse("build.jobs = 1").jobs, Some(1));
+}
```

Comment:
We should add coverage for the case where flags override the configuration.

Reply:
Added